__init__.py

```diff
@@ -68,22 +68,21 @@ def initialize(self):
         self.language = self.config_core.get('lang')
         self.load_vocab_files(join(dirname(__file__), 'vocab', self.lang))
         self.load_regex_files(join(dirname(__file__), 'regex', self.lang))
-        self.__build_switch_intent()
         self.__build_light_adjust_intent()
         self.__build_automation_intent()
-        self.__build_sensor_intent()
         self.__build_tracker_intent()
         self.register_intent_file(
             'set.climate.intent',
             self.handle_set_thermostat_intent
         )
+        #Working
```

Comment:
Can you remove this comment? It's a work in progress placeholder

Reply:
Yes, its just a note,

Reply:
done